p5/sketch/Skia2DRenderer/image.py

```diff
@@ -0,0 +1,118 @@
+from p5.core import constants
+from p5.core import PImage
+import numpy as np
+import skia
+import builtins
+
+
+class SkiaPImage(PImage):
+    def __init__(self, width, height, pixels=None):
+        self._width = width
+        self._height = height
+        self.pixels = (
+            pixels
+            if pixels is not None
+            else np.zeros((width, height, 4), dtype=np.uint8)
```

Comment:
I think it's useful to create a constant for 4 and name it something descriptive.

Reply:
What should I name it ? `RGBA_CHANNELS` ?
Do you have anything in mind?

Reply:
I think `RGBA_CHANNELS` works well!